### PR TITLE
UG Issues with regards to #247, #225, #224, #217

### PR DESCRIPTION
**Pull request by nicljr**
#247 -> Tag and Untag person event included examples

#225 -> Find parameter changed in UG

#224 -> 0 is a positive number but is not a valid number, so i changed for all the commands that have this INDEX line in their description

#217 -> I have fixed the optional tag for Name in edit contact

### CODE REVIEW

**Comment by nicljr** [comment]
> hansstanley

Updated the brackets and included race for find table

File: docs/UserGuide.md [hunk 1]
```diff
@@ -205,7 +205,7 @@ Use this command to favourite it so that you have easy access to it!
 _Here are some important requirements for you to take note:_
 
 * `INDEX` refers to the index of the contact you wish to edit in the current displayed list.
-  * `INDEX` must be a **positive integer**.
+  * `INDEX` must be a **positive integer** and (must be greater than 0).
```
Comment (igezt) [inline docs/UserGuide.md:208]:
```suggestion
  * `INDEX` must be a **positive integer** (must be greater than 0).
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -227,7 +227,7 @@ Use this command to unfavourite it!
...
```
Comment (igezt) [inline docs/UserGuide.md:230]:
```suggestion
  * `INDEX` must be a **positive integer** (must be greater than 0).
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -406,7 +408,7 @@ Use this command to expand and view more information about the selected contact.
 
 _Here are some important requirements for you to take note:_
 * `INDEX` refers to the index of the contact you wish to view more information on.
-    * `INDEX` must be a positive integer.
+    * `INDEX` must be a positive integer (must be greater than 0).
```
Comment (igezt) [inline docs/UserGuide.md:411]:
```suggestion
    * `INDEX` must be a **positive integer** (must be greater than 0). 
```

**Review by igezt — APPROVED** [review]
Nicely done! Just a few changes to keep the same format as the rest.

**Review by igezt — APPROVED** [review]
LGTM

**Review by igezt — APPROVED** [review]
LET US MERGE IT INNNNN

**Review by hansstanley — APPROVED** [review]

**Pull request merged**